rosdep/base.yaml

```diff
@@ -310,6 +310,27 @@ boost:
     xenial: [libboost-all-dev]
     yakkety: [libboost-all-dev]
     zesty: [libboost-all-dev]
+boost-program-options:
+  ubuntu:
+    bionic: [libboost-program-options1.65.1]
+    precise: [libboost-program-options1.46.1]
+    trusty: [libboost-program-options1.54.0]
+boost-program-options-dev:
```

Comment:
This looks reasonable, but should use the key that's the same as the debian/ubuntu key as that's our default name.

Reply:
alright.